README.md

```diff
@@ -0,0 +1,170 @@
+  <br>
+  <a href="https://delta-io.github.io/delta-rs/python/">Python docs</a>
+  ·
+  <a href="https://docs.rs/deltalake/latest/deltalake/">Rust docs</a>
+  ·
+  <a href="https://github.com/delta-io/delta-rs/issues/new?template=bug_report.md">Report a bug</a>
+  ·
+  <a href="https://github.com/delta-io/delta-rs/issues/new?template=feature_request.md">Request a feature</a>
+  ·
+  <a href="https://github.com/delta-io/delta-rs/issues/1128">Roadmap</a>
+  <br>
+  <br>
+  <a href="https://pypi.python.org/pypi/deltalake">
+    <img alt="Deltalake" src="https://img.shields.io/pypi/l/deltalake.svg?style=flat-square&color=00ADD4&logo=apache">
+  </a>
+  <a target="_blank" href="https://github.com/delta-io/delta-rs" style="background:none">
+    <img src="https://img.shields.io/github/stars/delta-io/delta-rs?logo=github&color=F75101">
+  </a>
+  <a target="_blank" href="https://crates.io/crates/deltalake" style="background:none">
+    <img alt="Crate" src="https://img.shields.io/crates/v/deltalake.svg?style=flat-square&color=00ADD4&logo=rust" >
+  </a>
+  <a href="https://pypi.python.org/pypi/deltalake">
+    <img alt="Deltalake" src="https://img.shields.io/pypi/v/deltalake.svg?style=flat-square&color=F75101&logo=pypi" >
+  </a>
+  <a href="https://pypi.python.org/pypi/deltalake">
+    <img alt="Deltalake" src="https://img.shields.io/pypi/pyversions/deltalake.svg?style=flat-square&color=00ADD4&logo=python">
+  </a>
+  <a target="_blank" href="https://go.delta.io/slack">
+    <img alt="#delta-rs in the Delta Lake Slack workspace" src="https://img.shields.io/badge/slack-delta-blue.svg?logo=slack&style=flat-square&color=F75101">
+  </a>
+</p>
+
+The delta-rs project aims to unlock the power of the Deltalake for as many users and projects as possible
+by providing native low level APIs aimed at developers and integrators, as well as a high level operations
+API that lets you query, inspect, and operate your Deltalake with ease.
+
+| Source                | Downloads                         | Installation Command    | Docs            |
+| --------------------- | --------------------------------- | ----------------------- | --------------- |
+| **[PyPi][pypi]**      | [![Downloads][pypi-dl]][pypi]     | `pip install deltalake` | [Docs][py-docs] |
+| **[Crates.io][pypi]** | [![Downloads][crates-dl]][crates] | `cargo add deltalake`   | [Docs][rs-docs] |
+
+[pypi]: https://pypi.org/project/deltalake/
+[pypi-dl]: https://img.shields.io/pypi/dm/deltalake?style=flat-square&color=00ADD4
+[py-docs]: https://delta-io.github.io/delta-rs/python/
+[rs-docs]: https://docs.rs/deltalake/latest/deltalake/
+[crates]: https://crates.io/crates/deltalake
+[crates-dl]: https://img.shields.io/crates/d/deltalake?color=F75101
+
+## Table of contents
+
+- [Quick Start](#quick-start)
+- [Integartions](#integrations)
+- [Features](#features)
+
+## Quick Start
+
+The deltalake library aim to adopt familiar patterns from other libraries in data processing,
```

Comment:
Suggest either "The Delta Lake library aims to..." or "The `deltalake` library aims to..."